TODO/why-our-website-is-faster-than-yours.md

```diff
@@ -2,7 +2,7 @@
 * 原文作者 : [by Declan](https://www.voorhoede.nl/en/contact/)
 * 译文出自 : [掘金翻译计划](https://github.com/xitu/gold-miner)
 * 译者 : [hpoenixf](https://github.com/hpoenixf)
-* 校对者: 
+* 校对者: [MAYDAY1993](https://github.com/MAYDAY1993),[https://github.com/circlelove](https://github.com/circlelove)
```

Comment:
你不要凶人家嘛 


发自 网易邮箱大师
在2016年08月12日 10:14，circlelove 写道:

In TODO/why-our-website-is-faster-than-yours.md:

> @@ -2,7 +2,7 @@
>  * 原文作者 : [by Declan](https://www.voorhoede.nl/en/contact/)
>  * 译文出自 : [掘金翻译计划](https://github.com/xitu/gold-miner)
>  * 译者 : [hpoenixf](https://github.com/hpoenixf)
> -* 校对者: 
> +* 校对者: [MAYDAY1993](https://github.com/MAYDAY1993),[https://github.com/circlelove](https://github.com/circlelove)


喂喂喂能不能认真点！ 我名字呢= =只有链接了//

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.